src/shammodels/ramses/src/modules/AMRGridRefinementHandler.cpp

```
@@ -661,9 +668,1910 @@ void shammodels::basegodunov::modules::AMRGridRefinementHandler<Tvec, TgridVec>:
+                      .get(id_patch)
+                      .get()
+                      .get_read_access(depends_list)),
+              cell_graph_zm(
+                  shambase::get_check_ref(storage.cell_graph_edge)
+                      .get_refs_dir(Direction::zm)
+                      .get(id_patch)
+                      .get()
+                      .get_read_access(depends_list))
+
+        {
+            p_id    = id_patch;
```

Cppcheck (2.21.0)

[performance] Variable 'p_id' is assigned in constructor body. Consider performing initialization in initialization list.

(useInitializationList)